x/supply/client/cli/query.go

```diff
@@ -0,0 +1,104 @@
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/supply/types"
+	"github.com/spf13/cobra"
+)
+
+// GetQueryCmd returns the cli query commands for this module
+func GetQueryCmd(cdc *codec.Codec) *cobra.Command {
+	// Group supply queries under a subcommand
+	supplyQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the supply module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	supplyQueryCmd.AddCommand(client.GetCommands(
+		GetCmdQueryTotalSupply(cdc),
+	)...)
+
+	return supplyQueryCmd
+}
+
+// GetCmdQueryTotalSupply implements the query total supply command.
+func GetCmdQueryTotalSupply(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "total [denom (optional)]",
```

Comment:
```suggestion
		Use:   "total [denom]",
```

It may possibly be a personal preference, but I reckon the `(optional)` is quite antiaesthetic.

Reply:
IMHO  the examples provided by the `help` command suffice